### PR TITLE
UTF-8 charset fix :P

### LINKED ISSUE #16: UFT-8 Support
Missing utf-8 support

### PR DESCRIPTION
**Pull request by Behoston**
resolve #16 
and probably #10

### CODE REVIEW

File: README [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ##########################################
 ##                                      ##
 ##      Bungee Chat Filter Config       ##
-##                 v1.7                 ##
+##                 v1.8                 ##
```
Comment (zedwick) [inline README]: Could you please not change the version number in your PR, thanks.

File: pom.xml [hunk 2]
```diff
@@ -6,7 +6,7 @@
 
     <groupId>BungeeChatFilter</groupId>
     <artifactId>BungeeChatFilter</artifactId>
-    <version>1.7</version>
+    <version>1.8</version>
```
Comment (zedwick) [inline pom.xml]: Don't change version in PR, please

File: src/main/resources/config.yml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ##########################################
 ##                                      ##
 ##      Bungee Chat Filter Config       ##
-##                 v1.7                 ##
+##                 v1.8                 ##
```
Comment (zedwick) [inline src/main/resources/config.yml]: Don't change version in PR, please

**Review by zedwick — CHANGES_REQUESTED** [review]
Thanks, I'll test that later.

Could you revert the version change in your PR though, please.

**Comment by Behoston** [comment]
Reverted

**Pull request merged**